### PR TITLE
Update hero

### LINKED ISSUE #197: China website banner - replace current one with a new version
Replace current banner with one from this doc https://drive.google.com/open?id=1dArR_6YqwF95kpuEyYeJiLqx8yF-Sc7W

Re-direct link https://www.brighttalk.com/webcast/6793/311297

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Change homepage hero

## QA

- Checkout feature branch
- `./run serve`
- View site <http://0.0.0.0:8018>
- Check hero is same as design <https://drive.google.com/open?id=1dArR_6YqwF95kpuEyYeJiLqx8yF-Sc7W>

## Issue / Card

Fixes #197

## Screenshots

hero

### CODE REVIEW

File: static/sass/custom/_ubuntu_intro.scss [hunk 1]
```diff
@@ -191,3 +191,32 @@
     width: 16px;
   }
 }
+
+.p-takeover {
+  background-image: url(https://assets.ubuntu.com/v1/f5b47308-china-banner-takeover-background.jpg);
+  background-position: center center;
+  background-size: cover;
+  background-color: #2C001E;
+  color: #fff;
```
Comment (anthonydillon) [inline static/sass/custom/_ubuntu_intro.scss]: Can you use `$color-x-light` instead please.

**Comment by steverydz** [comment]
@anthonydillon Changes made

**Comment by anasereijo** [comment]
@steverydz can you please replace ubuntu logo with the white version?
https://assets.ubuntu.com/v1/2922341c-ubuntu_white-orange_hex_su.svg

**Comment by steverydz** [comment]
@anasereijo Done

hero

**Comment by anasereijo** [comment]
@steverydz thanks

**Pull request merged**